### PR TITLE
Weather 

### PR DESCRIPTION
**Pull request by itsbrotherdan**

### CODE REVIEW

File: client/components/Event.jsx [hunk 1]
```diff
@@ -32,6 +42,10 @@ const Event = (props) => {
   const [going, setGoing] = useState(false);
   // const [eventCount, setEventCount] = useState(0);
 
+  const [weatherData, setWeatherData] = useState(null);
+
+  const API_URL = `https://api.open-meteo.com/v1/forecast?daily=weathercode&start_date=2023-04-10&end_date=2023-04-10&timezone=auto&latitude=${props.eventData.coordinates[1]}&longitude=${props.eventData.coordinates[0]}`;
```
Comment (copelka) [inline client/components/Event.jsx]: API calls should be made on the server side

**Review by copelka — CHANGES_REQUESTED** [review]
API calls should be made on the server side

**Review by matthewpmi — APPROVED** [review]
lgtm - nevermind

**Review by matthewpmi — APPROVED** [review]
wait!

**Review by opsparkbtc — APPROVED** [review]

**Review by kevinklein09 — APPROVED** [review]

**Review by matthewpmi — APPROVED** [review]
okay now this is good

**Review by kevinklein09 — APPROVED** [review]

**Pull request merged**